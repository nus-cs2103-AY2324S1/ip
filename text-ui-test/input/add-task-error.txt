todo do task 1
event event tday
event  /from 9pm /to 11pm
event  /from  /to 9pm
event talk /from  /to 
event talk /from 12am /to 
list
deadline  /from 9pm
deadline do sth /by 
deadline  /by 9pm
list
bye