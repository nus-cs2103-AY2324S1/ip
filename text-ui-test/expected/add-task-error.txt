____________________________________________________________
Loading data from hard disk ...
Done loading.
____________________________________________________________
Hello from Quack-NKN
What can I do for you?
____________________________________________________________
In: ____________________________________________________________
Got it. I've added this task:
[T][ ] do task 1
Now you have 1 in the list.
____________________________________________________________
In: ____________________________________________________________
Quack, keyword '/from' not found. It must be present for me to mark the start time!
____________________________________________________________
In: ____________________________________________________________
Quack, no empty event please!
____________________________________________________________
In: ____________________________________________________________
Quack, no empty event please!
____________________________________________________________
In: ____________________________________________________________
Quack, no empty start time please!
____________________________________________________________
In: ____________________________________________________________
Quack, no empty end time please!
____________________________________________________________
In: ____________________________________________________________
1. [T][ ] do task 1
____________________________________________________________
In: ____________________________________________________________
Quack, keyword '/by' not found.It must be present for me to mark the deadline time!
____________________________________________________________
In: ____________________________________________________________
Quack, no empty deadline time please!
____________________________________________________________
In: ____________________________________________________________
Quack, no empty deadline task please!
____________________________________________________________
In: ____________________________________________________________
1. [T][ ] do task 1
____________________________________________________________
In: ____________________________________________________________
Bye. Hope to see you again soon!
____________________________________________________________